diff --git a/boltzmann/camb/camb_interface.py b/boltzmann/camb/camb_interface.py
index 7d6236b..6effb64 100644
--- a/boltzmann/camb/camb_interface.py
+++ b/boltzmann/camb/camb_interface.py
@@ -167,8 +167,8 @@ def setup(options):
 Please use any these (separated by spaces): {}""".format(bad_power, good_power))
 
     # sigma(r) JTA+CZ
-    r_vec = vector_fron_config(options, "sigmar_rmin", "sigmar_rmax", "sigmar_dr", "sigmar_r")
-    more_config["sigma_r_vec"] = r_vec
+    # removed on Aug 13, 2023
+    # no need of sigma(r)
 
     camb.set_feedback_level(level=options.get_int(opt, "feedback", default=0))
     return [config, more_config]
@@ -195,6 +195,7 @@ def vector_from_config(options, vmin, vmax, dv, vec):
 #
 def save_sigmaR(r, p, block, more_config):
     r_vec = more_config["sigma_r_vec"]
+
     R, z, sigma_r = r.get_sigmaR(R=r_vec, return_R_z=True)
     block.put_grid("sigma_r", "z", z, "r", R, "sigma", sigma_r)
 
@@ -688,7 +689,10 @@ def execute(block, config):
 
     save_derived_parameters(r, p, block)
     save_distances(r, p, block, more_config)
-    save_sigmaR(r, p, block, more_config)
+
+    # removed on Aug 13, 2023
+    # no need of sigma(r)
+    #save_sigmaR(r, p, block, more_config)
 
     if p.WantTransfer:
         save_matter_power(r, p, block, more_config)
